.buildkite/hooks/pre-command

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+source buildkite-scripts/utils/logging.sh
+
+set -eu
+
+echo "--- :checkered_flag: Running pre-command for command ${BUILDKITE_COMMAND}"
+
+if [[ "${BUILDKITE_LABEL}" = *"Tests"* ]]; then
+  # We do this to prevent BK from uploading artifacts to s3
```

Comment:
Why not remove the artifacts from the buildkite step?
`artifact_paths: 'e2e/reports/*.html'`

Reply:
not sure what do you mean?

the pre-command is to make *.html test reports available in BK directly, otherwise we cannot read it from BK Web UI.

I do want to keep it available in BK.